triage/labels.rst

```diff
@@ -10,405 +10,135 @@ Many labels are shared for both use cases, while some are dedicated
-:ref:`release schedule <branchstatus>` is decided by the release manager.
-Triagers may recommend this priority and should notify the release manager by
-tagging them in a comment using ``@username``. If needed, consult the
-:ref:`release schedule <branchstatus>` and the release's associated PEP for the
-release manager's name.
-
-Keywords
---------
-Various informational flags about the issue. Multiple values are possible.
-
-+---------------+------------------------------------------------------------+
-|    Keyword    |                        Description                         |
-+===============+============================================================+
-| easy          | Fixing the issue should not take longer than a day for     |
-|               | someone new to contributing to Python to solve.            |
-+---------------+------------------------------------------------------------+
-
-Nosy List
----------
-A list of people who may be interested in an issue.
-
-This used to be a feature of the old issue tracker. On GitHub issues the
-same effect is achieved by tagging people in a comment using ``@username``.
-
-It is acceptable to tag someone to if you think the issue should be brought to
-their attention. Use the :ref:`experts` to know who wants to be added to the
-nosy list for issues targeting specific areas.
-
-If you want to subscribe yourself to an issue, click the *🔔 Subscribe*
-button in the sidebar. Similarly, if you were tagged by somebody else but
-decided this issue is not for you, you might click the *🔕 Unsubscribe*
-button in the sidebar.
-
-Assignees
----------
-Who is expected to take the next step in resolving the issue.
-
-It is acceptable to assign an issue to someone if the issue cannot move
-forward without their help, e.g., they need to make a technical decision to
-allow the issue to move forward. Also consult the :ref:`experts` as certain
-stdlib modules should always be assigned to a specific person.
-
-Note that in order to assign an issue to someone, that person **must** be
-a team member, likely a Triager or a core developer.
-
-Dependencies
-------------
-The issue requires the listed issue(s) to be resolved first before it can move
-forward. This is achieved using checkbox lists in the initial issue description
-comment. Long story short, if you add this::
-
-    - [x] #739
-    - [ ] https://github.com/octo-org/octo-repo/issues/740
-    - [ ] Add delight to the experience when all tasks are complete :tada:
-
-then those will become sub-tasks on the given issue. Moreover, GitHub will
-automatically mark a task as complete if the other referenced issue is
-closed.
-
-More details in the `official GitHub documentation
-<https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists>`_.
-
-Superseder
-----------
-The issue is a duplicate of the listed issue(s). To make GitHub mark
-an issue as duplicate, write "Duplicate of #xxxx" in a comment.
```

Comment:
These have been adapted and moved to the `issue-tracker.rst` page.

Reply:
For these that have been moved to another page, is it worth preserving the `#anchors` by putting them at the bottom, like we have on these other pages?

https://devguide.python.org/documentation/start-documenting/#style-guide
https://devguide.python.org/documentation/start-documenting/#translating

```rst
Style Guide
===========

Moved to :doc:`style-guide`


Translating
===========

Moved to :doc:`translating`
```

https://devguide.python.org/#status-of-python-branches

```rst
Status of Python branches
-------------------------

Moved to :ref:`versions`
```